fb/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @comment = current_user.comments.build(comment_params)
+    if @comment.save
+      redirect_back(fallback_location: root_path)
+    else
+      render :new
```

Comment:
submitting a blank comment breaks the app:
- remember comments is embedded on that page
- also comments can become a lot, lets add a link on the post to go to a show page
![Screenshot 2019-09-10 at 5 31 28 PM](https://user-images.githubusercontent.com/10922508/64632492-0ff7fc00-d3f1-11e9-87ef-74879f6a02af.png)

Reply:
Good catch 🙂

Reply:
Thanks Sunny. This definitely needs to be fixed. I will implement the changes then submit another request.